### PR TITLE
HTTP STATUS and Handler

### PR DESCRIPTION
**Pull request by harshsuccessive**
Created a middleware
Created two files errorHandler.ts and notFoundRoute.ts
It shows the following error
{
error: "Not Found",
message: "error",
status: 500,
timestamp: "2019-01-08T17:38:21.929Z"
}

### CODE REVIEW

File: src/libs/routes/notFoundRoute.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from 'express'
+
+export default (req: Request, res: Response, next: NextFunction) =>{
```
Comment (namanpar) [inline src/libs/routes/notFoundRoute.ts:3]: Write proper name for function

File: src/libs/routes/errorHandler.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Request, Response, NextFunction } from 'express';
+
+export default (err, req: Request, res: Response, next: NextFunction ) =>{
```
Comment (namanpar) [inline src/libs/routes/errorHandler.ts:3]: Write proper name for function

**Review by namanpar — CHANGES_REQUESTED** [review]
You should write proper name for functions. It is a good practice to export nameless function

File: src/libs/routes/notFoundRoute.ts (hunk 1 above)
Comment (harshsuccessive) [reply src/libs/routes/notFoundRoute.ts]: Wrote the Name please do check

File: src/libs/routes/errorHandler.ts (hunk 2 above)
Comment (harshsuccessive) [reply src/libs/routes/errorHandler.ts]: Wrote the name please do check

**Review by namanpar — APPROVED** [review]

**Pull request merged**